README.md

```diff
@@ -529,6 +529,8 @@ Update your README.md to embed your metrics:
 ![Metrics](https://github.com/my-github-user/my-github-user/blob/master/github-metrics.svg)
 <!-- If you're using "main" as default branch -->
 ![Metrics](https://github.com/my-github-user/my-github-user/blob/main/github-metrics.svg)
+<!-- If you're using the "columns" display mode -->
+<img src="https://github.com/my-github-user/my-github-user/blob/master/github-metrics.svg" alt="Metrics" width="100%">
```

Comment:
Can you patch [`.github/readme/partials/setup/action/setup.md`](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/setup/action/setup.md) with this change instead? The readme is now auto-generated from these files so changes won't be reflected when the actions bot rebuild it

Reply:
Thanks, I didn't know it was rebuilt ! It's done